mbuild/lattice.py

```diff
@@ -308,7 +308,7 @@ def _validate_lattice_points(self, lattice_points):
                                      "None was passed in as position for {}."
                                      .format(name))
                 for coord in pos:
-                    if (0 > coord >=1):
+                    if (coord is None) or (0 > coord) or (coord >=1):
```

Comment:
I believe you can write this as `0 > coord >= 1`

Reply:
It seems as if python treats this as an AND statement. I was not able to get that to work.

Reply:
Actually, it seems like it just shorts out and ends at the first test.

Reply:
Python does short-circuit on conditional statements.

And sorry I misread the statement what I wrote is not equivalent.